### PR TITLE
Issue PS-1484 feat: FIx UI issues

### PR DESCRIPTION
**Pull request by AkshataKatwal16**

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -84,7 +84,8 @@
     "NEW_FACILITATOR": "New Facilitator",
     "SEARCHBAR_PLACEHOLDER": "Search Facilitators..",
     "FACILITATOR_CREATED_SUCCESSFULLY": "Teacher has been Successfully Created",
-    "FACILITATOR_UPDATED_SUCCESSFULLY": "Teacher has been Successfully Updated!"
+    "FACILITATOR_UPDATED_SUCCESSFULLY": "Teacher has been Successfully Updated!",
```
Comment (itsvick) [inline public/locales/en/common.json:87]: @AkshataKatwal16 keep first letter of the sentence as capital rest in small case

File: src/components/AddTeamLeaderModal.tsx [hunk 2]
```diff
@@ -310,7 +310,7 @@ const AddTeamLeaderModal: React.FC<AddLearnerModalProps> = ({
         open={open}
         onClose={onClose}
         showFooter={false}
-        modalTitle={t("TEAM_LEADERS.NEW_TEAM_LEADER")}
+        modalTitle={isEditModal? t("TEAM_LEADERS.NEW_TEAM_LEADER"): t("TEAM_LEADERS.EDIT_TEAM_LEADERS")}
```
Comment (itsvick) [inline src/components/AddTeamLeaderModal.tsx:313]: @AkshataKatwal16 it should be EDIT_TEAM_LEADER

**Review by itsvick — CHANGES_REQUESTED** [review]

File: public/locales/en/common.json (hunk 1 above)
Comment (AkshataKatwal16) [reply public/locales/en/common.json]: updated

File: src/components/AddTeamLeaderModal.tsx (hunk 2 above)
Comment (AkshataKatwal16) [reply src/components/AddTeamLeaderModal.tsx]: updated

File: src/pages/logout.tsx [hunk 3]
```diff
@@ -21,6 +21,8 @@ function Logout() {
     };
     userLogout();
     localStorage.removeItem("token");
+    localStorage.removeItem("preferredLanguage");
```
Comment (itsvick) [inline src/pages/logout.tsx:24]: @AkshataKatwal16 dont remove the preferred language on logout

**Review by itsvick — CHANGES_REQUESTED** [review]

File: src/pages/logout.tsx (hunk 3 above)
Comment (AkshataKatwal16) [reply src/pages/logout.tsx]: okay...as per sanket suggestion i have removed that,now reverted back that line

File: public/locales/en/common.json [hunk 4]
```diff
@@ -87,8 +87,9 @@
     "ALL_BLOCKS": "Blocks",
     "NEW_FACILITATOR": "New Facilitator",
     "SEARCHBAR_PLACEHOLDER": "Search Facilitators..",
-    "FACILITATOR_CREATED_SUCCESSFULLY": "Teacher has been Successfully Created",
-    "FACILITATOR_UPDATED_SUCCESSFULLY": "Teacher has been Successfully Updated!"
+    "FACILITATOR_CREATED_SUCCESSFULLY": "Teacher has been successfully sreated",
```
Comment (itsvick) [inline public/locales/en/common.json:90]: @AkshataKatwal16 typo here

Comment (AkshataKatwal16) [reply public/locales/en/common.json]: fixed

**Pull request merged**